mautic/mautic
# Style and Script tags are allowed in the Email/Page HTMLs

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | N
| BC breaks?    | N
| Deprecations? | N
| Fixed issues  |  #1959 

## Description

**Warning:** https://github.com/mautic/mautic/pull/1962 and https://github.com/mautic/mautic/pull/1992 must be merged first in order to test this PR successfully.

The full-page Froala editor was removing `<style/>` and `<script/>` tags which caused a big issues as discussed in #1959. After discussion with Froala guys I found out there is another option which removes them (https://github.com/froala/wysiwyg-editor/issues/1396). This PR clears this option.

## Steps to reproduce the bug (if applicable)
1. Create an email or a page.
2. Save it and review the code. You will notice that all the style tags are gone.

## Steps to test this PR
1. Apply this PR
2. Rebuild prod assets with `app/console mautic:assets:generate` command
3. Try again (create a new page/email, because the old one won't get fixed). The style tags should remain.
